src/web/mjs/connectors/DigitalMargaret.mjs

```diff
@@ -0,0 +1,43 @@
+import SpeedBinb from './templates/SpeedBinb.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class DigitalMargaret extends SpeedBinb {
+
+    constructor() {
+        super();
+        super.id = 'digitalmargaret';
+        super.label = 'デジタルマーガレット (Digital Margaret)';
+        this.tags = ['manga', 'japanese'];
+        this.url = 'https://digitalmargaret.jp';
+    }
+
+    async _getMangaFromURI(uri) {
+        const request = new Request(uri, this.requestOptions);
+        const [data] = await this.fetchDOM(request, 'section#product div.content h3');
+        const title = data.textContent.trim();
+        const id = uri.pathname;
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        const request = new Request(new URL('/', this.url), this.requestOptions);
+        const data = await this.fetchDOM(request, 'section#serial ul.serial-list li a');
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, this.url),
+                title: element.querySelector('img').alt
```

Comment:
Got error because alt attribute doesnt always exist. There is a h3 a under the picture that seems to got the same tilte and the url btw. Its more reliable maybe.

Reply:
img tags were converted to source tags. Fixed.
(Elements with h3 tags are not manga listings)